src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -887,6 +1204,54 @@ public ListBoxModel doFillCheckoutCredentialsIdItems(@AncestorInPath SCMSourceOw
                     // TODO add support for tags and maybe feature branch identification
             };
         }
+
+        public List<NamedArrayList<? extends SCMSourceTraitDescriptor>> getTraitsDescriptorLists() {
+            List<SCMSourceTraitDescriptor> all =
```

Comment:
Not easily. There is some grouping and ordering going on as well as type specific filtering.
I'll probably introduce some helper methods into scm-api once we see some more patterns of usage fall out